openfl/component/aggregator/aggregator.py

```diff
@@ -341,9 +341,9 @@ def get_aggregated_tensor(self, collaborator_name, tensor_name,
         #  The returned tensor will
         # be recompressed anyway.
         if 'compressed' in tags:
-            tags.remove('compressed')
+            list(tags).remove('compressed')
```

Comment:
`list(tags)` returns new list and `'compressed'` will be removed from that list, not form original `tags`.

Reply:
Thanks @aleksandr-mokrov. Fixed.